lib/elixir/lib/calendar/iso.ex

```diff
@@ -788,12 +832,17 @@ defmodule Calendar.ISO do
   @spec date_to_string(year, month, day, :basic | :extended) :: String.t()
   @impl true
   def date_to_string(year, month, day, format \\ :extended)
+      # when is_year(year) and is_month(month) and is_day(day) and format in [:basic, :extended] do
```

Comment:
I think @josevalim is talking about this commented out code.

Reply:
thank you. comment removed